covalent_ssh_plugin/ssh.py

```diff
@@ -23,25 +23,24 @@
 """
 
 # Required for all executor plugins
-import io
 import os
 import socket
 from contextlib import redirect_stderr, redirect_stdout
 from multiprocessing import Queue as MPQ
```

Comment:
```suggestion
from multiprocessing import Queue as MPQ
```
Is this still relevant?

Reply:
Its not, neither are several unused imports which are peppered everywhere. I'll remove them as well.